.circleci/config.yml

```diff
@@ -49,6 +49,17 @@ workflows:
 
 version: 2
 jobs:
+  validate-config:
+    docker:
+      - image: node
+    steps:
+      - run:
+        - >
+          if ! eMSG=$(circleci config validate -c .circleci/config.yml); then
```

Comment:
We should share the command between this and the husky commit hook so we're not repeating ourselves.